src/task/script.js

```diff
@@ -147,10 +148,12 @@ taskForm.onsubmit = async (e) => {
       neelam: Number(lossRateNeelam),
     },
     isNoteworthy: isNoteworthy == 'on',
+    estimatedNoOfDays: Number(estimatedNoOfDays),
   };
 
   if (status === 'ASSIGNED') {
     dataToBeSent.startedOn = new Date() / 1000;
+    delete dataToBeSent.estimatedNoOfDays;
```

Comment:
I didn't get why we are deleting estimatedNoOfDays. Can you provide me with an overview of this?

Reply:
because we are not supposed to send this data when `STATUS = AVAILABLE`. 
I kept this in consistent with pre-existing code.

Reply:
Ok, this makes sense.

Reply:
@ankushdharkar 
Created a issue ticket for this - https://github.com/Real-Dev-Squad/website-dashboard/issues/173.
Will raise PR for the same.